javascript/code-challenges/tree/__tests__/tree.test.js

```diff
@@ -61,4 +76,24 @@ describe('Binary Search Tree', () => {
         tree.add(2);
         expect(tree.findMaximumValue()).toBe(2);
     })
+
+    // this test fails because it recieves [5, 2, 11, 11, 7]
+    // am currently unsure why
+    it('Can successfully perform a breadth first search', () => {
+        tree.add(5);
+        tree.add(11);
```

Comment:
I am not sure why it is adding 11 twice, I would say that I think it has to be coming from the breadth first method because it isn't happening anywhere else you are using the add method.